cognite/client/data_classes/cdm/__init__.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from cognite.client.config import global_config
+from cognite.client.utils._experimental import FeaturePreviewWarning
+
+FeaturePreviewWarning("alpha", "alpha", "Core Data Model").warn()
+
+if global_config.cdm_model_version == "v1":
```

Comment:
I don't think we should do this. In migration scenarios it will be useful to import multiple versions at the same time. And I don't really think it's worth introducing complexity just to avoid being explicit about which version to import.

Reply:
(explicit is good!)